src/node_env_var.cc

```diff
@@ -62,7 +62,26 @@ class MapKVStore final : public KVStore {
 
  private:
   mutable Mutex mutex_;
+#ifdef _WIN32
+  struct Hash {
+    inline size_t operator()(const std::string& str) const {
+      return std::hash<std::string>()(ToLower(str));
+    }
+  };
+
+  struct Equal {
+    inline bool operator()(const std::string& a, const std::string& b) const {
+      return a.size() == b.size() &&
+             std::equal(a.begin(), a.end(), b.begin(), [](char a, char b) {
+               return ToLower(a) == ToLower(b);
```

Comment:
I recommend @lemire's blog post about this: https://lemire.me/blog/2020/04/30/for-case-insensitive-string-comparisons-avoid-char-by-char-functions

Reply:
https://github.com/nodejs/node/blob/d396a041f71cc055ad60b0abc63ad81c0ee6a574/src/util.h#L356-L357

Thanks, but `strncasecmp()` seems to have a locale problem. Simplified it using `StringEqualNoCase`. PTAL.